shoplist/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListResponse(
+    val list_id: String,
+    val userId: Int,
+    val listName: String
+)
```

Comment:
```suggestion
data class ShopListResponse(
    @SerializedName("list_id")val list_id: String,
    @SerializedName("userId")val userId: Int,
    @SerializedName("listName")val listName: String
)
```

You have to serialize the data model that you are using for the endpoint.